### PR TITLE
fall back to sendmail if smtp server is disabled

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: lib/sendMailTransport.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const nodemailer = require('nodemailer');
+
+const sendmail = {
```
Comment (VoyTechnology) [inline lib/sendMailTransport.js]: I would prefer if it was part of `_config.yml` as `emailFallback` or something like that.

**Comment by butlerx** [comment]
I Considered that but I didn't see that as a fallback but as an alternative

**Comment by VoyTechnology** [comment]
But it is a fallback. It would be an alternative if you had an option to use one or another. But here the sendmail is only used if SMTP fails.

**Comment by SeanHealy33** [comment]
Why is there yarn update's in this?

**Comment by butlerx** [comment]
I had some weird dependency issue I solved with an upgrade. Can remove it if yiu want

**Comment by butlerx** [comment]
we dont want it as an alternative, we only want it for when smtp fails

**Comment by VoyTechnology** [comment]
What is the point of reviews if you ignore them.

**Comment by butlerx** [comment]
i didnt ignore it had a green button

**Pull request merged**